1-Introduction/1-intro-to-ML/translations/README.es.md

```diff
@@ -0,0 +1,113 @@
+
+## Lo que aprenderás en el curso
+
+En este currículum, vamos a cubrir solo los conceptos clave de machine learning que un principiante debería conocer. Cubrimos algo a lo que le llamamos "machine learning clásico" usando principalmente Scikit-learn, una biblioteca excelente que muchos estudiantes utilizan para aprender las bases. Para entender conteptos más amplios de la inteligencia artificial o deep learning, es indispensable tener un fuerte conocimiento de los fundamentos, y eso es lo que nos gustaría ofrecerte aquí.
+
+En este curso aprenderás:
+
+- conceptos clave del machine learning
+- la historia de ML
+- la justicia y el ML
+- técnicas de regresión en ML
+- técnicas de clasificación en ML
+- técnicas de agrupamiento en ML
+- técnicas de procesamiento del lenguaje natural en ML
+- técnicas de previsión de series temporales en ML
+- reforzamiento del aprendizaje
+- ML aplicada al mundo real
+
+## Lo que no cubriremos
+
+- deep learning
+- redes neuronales
+- inteligencia artificial (IA)
+
+Para tener una mejor experiencia de aprendizaje, vamos a evitar las complejidades de las redes neuronales, "deep learning" (construcción de modelos de muchas capas utilizando las redes neuronales) e inteligencia artificial, que se discutirá en un currículum diferente. En un futuro también ofreceremos un currículum acerca de la ciencia de datos para enfocarnos en ese aspecto de este campo.
+
+## ¿Por qué estudiar machine learning?
+
+Machine learning, desde una perspectiva de los sistemas, se define como la creación de sistemas automáticos que pueden aprender patrones ocultos a partir de datos para ayudar en tomar decisiones inteligentes.
+
+Esta motivación está algo inspirada por como el cerebro humano aprende ciertas cosas basadas en los datos que percibe en el mundo real.
+
+✅ Piensa por un minuto en porqué querría un negocio intentar implementar estrategias de machine learning vs. programar un motor basado en reglas.
+
+### Aplicaciones del machine learning
+
+Las aplicaciones del machine learning hoy en día están casi en todas partes, y son tan ubicuas como los datos que fluyen alrededor de nuestras sociedades, generados por nuestros teléfonos inteligentes, dispositivos conectados a internet, y otros sistemas. Considerando el inmenso potencial de los algoritmos estado del arte de machine learning, investigadores han estado explorando su capacidad de resolver problemas multidimensionales y multidisciplinarios de la vida real con resultados muy positivos.
+
+**Tú puedes utilizar machine learning de muchas formas**:
+
+- Para predecir la probabilidad de enfermedad a partir del historial médico o reportes de un paciente.
+- Para aprovechar datos del clima y predecir eventos climatológicos.
+- Para entender la intención de un texto.
```

Comment:
_intencion_ reads tot me like _intent_. I think _connotacion_ or _sentido_ would be better, since the expectation is that you can determine if something is positive or negative, it tends to read better, like:

> el texto tiene una connotacion positiva

or

> el texto tiene un sentido negativo